### ISSUE TITLE
pachi cant find caffe/caffe.hpp

### ISSUE THREAD

**Issue by JDJoe** [pasky/pachi#54]
caffe is compiled and files are in:

/something/PACHI_GO_ENGINE/caffe/distribute/include/caffe/caffe.hpp
/something/PACHI_GO_ENGINE/caffe/include/caffe/caffe.hpp

compiling pachi in: 

`/something/PACHI_GO_ENGINE/pachi/`

I updated the Makefile :

```
DCNN=1
#CAFFE_LIB=/usr/local/lib
CAFFE_LIB=/something/PACHI_GO_ENGINE/caffe/include  #(and few other versions of this)
```

and I still get dcnn.cpp:13:27: fatal error: caffe/caffe.hpp: No such file or directory

```
[CC]   fbook.c
[CC]   chat.c
[CXX]   dcnn.cpp
dcnn.cpp:13:27: fatal error: caffe/caffe.hpp: No such file or directory
compilation terminated.
Makefile.lib:78: recipe for target 'dcnn.o' failed
make: *** [dcnn.o] Error 1
```

**Comment by lemonsqueeze** [comment]
Yes, Makefile needs to pass the path to caffe includes.

Try with this [Makefile](https://github.com/pasky/pachi/files/404123/Makefile.txt), setting CAFFE_PREFIX to `/something/PACHI_GO_ENGINE/caffe/distribute/`
or install caffe somewhere where compiler will find it (/usr/local for instance)

**Comment by pasky** [comment]
Closed by PR #55, thank you!
